core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -491,8 +491,9 @@ private void syncInodeMetadata(LockedInodePath inodePath)
       InterruptedException {
     if (!inodePath.fullPathExists()) {
       loadMetadataForPath(inodePath);
+    } else {
+      syncExistingInodeMetadata(inodePath);
```

Comment:
if a path does not exist in alluxio yet, load metadata should be equivalent to loading + syncing.

Reply:
Can you add this as a comment?